core/dns.go

```diff
@@ -14,6 +14,30 @@ import (
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/miekg/dns"
 )
 
+const (
+	// NoAddrFilter is used to tell LookupHost to query both A and AAAA records
+	NoAddrFilter = iota
```

Comment:
give a type to this:

type AddrFilter int

const (
    NoAddrFilter AddrFilter = iota
    IPv4OnlyFilter
    IPv6OnlyFilter
)

And then LookupHost gets a AddrFilter type in its params, not an int